static/app/views/organizationGroupDetails/grouping/errorMessage.tsx

```diff
@@ -52,7 +52,9 @@ function ErrorMessage({error, groupId, onRetry, orgSlug, projSlug}: Props) {
         };
       case 'missing_feature':
         return {
-          title: t('This project does not have the grouping tree feature'),
+          title: t(
+            'This project does not have the grouping breakdown available. Is your organization still an early adopter?'
```

Comment:
We will link to docs here once docs are live, right?

Reply:
yeah that's the idea